pkg/subctl/cmd/join.go

```diff
@@ -32,5 +46,8 @@ var joinCmd = &cobra.Command{
 		fmt.Printf("* Deploying the submariner operator\n")
 		err = install.Ensure(config, OperatorNamespace, operatorImage)
 		panicOnError(err)
+
+		fmt.Printf("* Deploying Submariner\n")
+		err = deploy.Ensure(config, "submariner-k8s-broker", SubmarinerNamespace, clusterID, serviceCIDR, clusterCIDR, subctlData)
```

Comment:
nit: use the constant for the Namespace instead?

Reply:
It turns out we don’t need it, it’s stored in `subctlData`!